src/include/daos_errno.h

```diff
@@ -285,7 +285,9 @@ extern "C" {
 	ACTION(DER_UPDATE_AGAIN,	(DER_ERR_DAOS_BASE + 41),	\
 	       update again)						\
 	ACTION(DER_NVME_IO,		(DER_ERR_DAOS_BASE + 42),	\
-	       NVMe I/O error)
+	       NVMe I/O error)						\
+	ACTION(DER_NO_CERT,		(DER_ERR_DAOS_BASE + 43),	\
+	       Unable to access one or more certificates)
```

Comment:
I was a little confused by this... This errno is eventually returned to the client application if the credentials are unable to be verified on the server side, right?

Reply:
No, this one is for the case where the cert can't be loaded on the agent side at all. I'm working on another PR for the invalid/incompatible cert case.

Reply:
Ah, ok. But the point is that the errno will be returned to the client application, right? i.e. We're not adding a new errno just for control plane tests?

Reply:
Yes, it is returned to the client application.